bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql

```diff
@@ -7,7 +7,8 @@ WITH
         source_table,
         "all_combos",
         cubed_attributes,
-        attribute_combinations
+        attribute_combinations,
+        add_windows_release_sample = channel == "release"
```

Comment:
Changed the logic here, the macro now only needs a boolean to indicate if the sampled column should be included